docs/openai-api.md

```
@@ -0,0 +1,408 @@
+<div align="center">
+
+# OpenAI API Documentation
+
+*Complete OpenAI API compatibility for local MLX inference*
+
+[Installation](#-installation--setup) • [Quick Start](#-basic-usage) • [Endpoints](#-supported-endpoints) • [Advanced Usage](#-advanced-usage)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid

(MD051, link-fragments)